### PR TITLE
 Primary and Destructive action buttons implementation completed.

### PR DESCRIPTION
**Pull request by Surezt22**
Primary and Destructive action buttons implementation completed.

The priority of the background colors is given to host-config.

### CODE REVIEW

File: source/reactnative/src/utils/enums.js [hunk 1]
```diff
@@ -131,3 +131,9 @@ export const Height = Object.freeze({
 	Stretch: 1
 });
 
+export const Sentiment = Object.freeze({
```
Comment (BalajiR) [inline source/reactnative/src/utils/enums.js]: Code indentation.

File: source/reactnative/src/styles/style-config.js [hunk 2]
```diff
@@ -70,8 +70,11 @@ export class StyleManager {
 		inputBorderWidth:{
 			borderWidth:this.themeConfig.input[Platform.OS].borderWidth,
 		},
-		defaultDestructiveForegroundColor:{
-			color: this.hostConfig.containerStyles.default.foregroundColors.attention.default,
+		defaultPositiveButtonBackgroundColor:{
+            backgroundColor: this.hostConfig.containerStyles.default.foregroundColors.attention.default,
+        },
+		defaultDestructiveButtonForegroundColor:{
+				color: this.hostConfig.containerStyles.default.foregroundColors.attention.default,
```
Comment (BalajiR) [inline source/reactnative/src/styles/style-config.js]: Code indentation.

File: source/reactnative/src/styles/style-config.js [hunk 3]
```diff
@@ -70,8 +70,11 @@ export class StyleManager {
 		inputBorderWidth:{
 			borderWidth:this.themeConfig.input[Platform.OS].borderWidth,
 		},
-		defaultDestructiveForegroundColor:{
-			color: this.hostConfig.containerStyles.default.foregroundColors.attention.default,
+		defaultPositiveButtonBackgroundColor:{
+            backgroundColor: this.hostConfig.containerStyles.default.foregroundColors.attention.default,
```
Comment (BalajiR) [inline source/reactnative/src/styles/style-config.js]: `positive` sentiment suppose to use `accent` color right ?

File: source/reactnative/src/styles/style-config.js (hunk 2 above)
Comment (Surezt22) [reply source/reactnative/src/styles/style-config.js]: Done

File: source/reactnative/src/styles/style-config.js (hunk 3 above)
Comment (Surezt22) [reply source/reactnative/src/styles/style-config.js]: yes. Changed to accent color

**Pull request merged**